### DISCUSSION TITLE
Custom Section Libraries

### DISCUSSION THREAD

**Discussion by dr13377** [idea-statica/ideastatica-public#159] [Features and Ideas]
It would be good to be able to create and save custom section libraries that could be used across multiple projects. For example the standard Australian Welded Columns (WC) and Welded Beams (WB) are not included in the supplied libraries.

**Comment by nathanlukeIDEA** [comment]
Thanks for posting your comment. Yes, currently there is no MPRL table available with welded cross-sections. We have received this request and hopefully should make this available sometime in the future.
